bindata/network/ovn-kubernetes/common/004-rbac-control-plane.yaml

```diff
@@ -71,6 +71,19 @@ rules:
   - patch
   - update
   - watch
+{{- if .DNS_NAME_RESOLVER_ENABLE }}
+- apiGroups: ["network.openshift.io"]
+  resources:
+  - dnsnameresolvers
+  verbs:
+  - create
```

Comment:
Does the control-plane really require all of these verbs?

Reply:
I think so, because cluster manager creates and deletes these objects (similar to cloudprivateipconfigs I suppose)